src/styles/main.scss

```diff
@@ -0,0 +1,61 @@
+@import './utils/variables';
```

Comment:
Ensure that the file `./utils/variables` exists and contains the necessary variable definitions, such as `$width-calendar`, `$width-square`, `$week-day`, and `$month-length`. Without these definitions, the SCSS will not compile correctly.